dfs/nfs/server_test.go

```diff
@@ -232,3 +246,57 @@ func TestWriteExports(t *testing.T) {
 	testWriteExports(preamble+conflict1+dummyBlock+conflict2+postamble,
 		preamble+etcExportsRemoveComment+conflict1+exportBlock+etcExportsRemoveComment+conflict2+postamble)
 }
+
+func TestRemoveDeprecated(t *testing.T) {
+	//Create a Server, then call cleanupBindMounts after lining up ducks
+	emptyTempDir, err := ioutil.TempDir("", "nfs_unit_tests_")
+	if err != nil {
+		t.Fatalf("unexpected error: %s", err)
+	}
+	defer os.RemoveAll(emptyTempDir)
+	t.Logf("created temp dir: %s", emptyTempDir)
+
+	nonemptyTempDir, err := ioutil.TempDir("", "nfs_unit_tests_")
+	if err != nil {
+		t.Fatalf("unexpected error: %s", err)
+	}
+	defer os.RemoveAll(nonemptyTempDir)
+
+	tmpfile, err := ioutil.TempFile(nonemptyTempDir, "foobar")
+	if err != nil {
+		t.Fatalf("unexpected error: %s", err)
+	}
+	defer os.Remove(tmpfile.Name())
+	t.Logf("created temp dir: %s", nonemptyTempDir)
+
+	// neuter NFS stop during tests
+	stop = func() error {
+		return nil
+	}
+	defer func() {
+		stop = stopImpl
+	}()
+
+	s1 := Server{
+		network:      "1.2.3.4/8",
+		basePath:     path.Join(emptyTempDir, "baseDir"),
+		exportedName: "foobar",
+	}
+
+	// deprecated dir is empty, deleted with no problem
+	assertPathExists(t, emptyTempDir)
```

Comment:
This doesn't hurt anything, but doesn't seem necessary as it basically verifies that `ioutil.TempDir()` works as advertised.